Makefile

```diff
@@ -26,7 +26,7 @@ $(TOOLS):
 	@mkdir -p $@
 $(TOOLS)/%: | $(TOOLS)
 	cd $(TOOLS_MOD_DIR) && \
-	$(GOCMD) build -o $@ $(PACKAGE)
+	$(GOCMD) build  -buildvcs=false -o $@ $(PACKAGE)
```

Comment:
I opened https://github.com/open-telemetry/opentelemetry-go-instrumentation/issues/814 to describe this a bit more, but I couldn't get `make docker-offsets` to work without it. 

Maybe I could make some permissions changes locally, but I think I have a pretty standard git setup. Since this should almost always be run in a container, it should work as the default path.

I don't think there's any benefit to VCS stamping for our tools, so I don't see a downside to adding it. What do you think?

Reply:
Tracking with #814. Going to resolve this.